cypress/e2e/cards.cy.ts

```diff
@@ -3,19 +3,68 @@ describe('in my home page', () => {
     cy.visit('http://localhost:4200/');
   });
 
-  it('Finds the "Previous" link', () => {
-    cy.contains('Previous');
+  it('has disabled "Previous" link in the fist page', () => {
+    cy.visit('/');
+    const findInPage = () => {
+      cy.get('li:has(span:contains(Previous))').then((element) => {
```

Comment:
That's a very interesting approach to recursively pushing a button until it is disabled.

![seal of approval](https://user-images.githubusercontent.com/1997886/79337932-53ce5900-7f1e-11ea-9f9c-e687035d7ae1.gif)

Reply:
you are so funny I love it hahaha